quam_builder/architecture/quantum_dots/README.md

```diff
@@ -0,0 +1,682 @@
+
+Subsequently, this document introduces the **VirtualGateSet** and **VirtualisationLayer** components. These extend the physical gate control capabilities provided by `GateSet` and `VoltageSequence` by adding one or more layers of virtual gates.
+
+Virtual gates simplify complex tuning procedures in experiments, especially for spin qubits, by allowing control over abstract parameters that map to multiple physical gate voltages.
+
+**`VirtualGateSet` builds upon `GateSet` and inherits all of its features**, including physical channel management, `VoltageTuningPoint` definitions, and all voltage control methods. This means you can use virtual gates while still having access to all the voltage control capabilities of the underlying `GateSet`.
+
+## 2. Overview and Workflow
+
+### 2.1. Components Overview
+
+#### 2.1.1 GateSet
+
+`GateSet` is a `QuantumComponent` grouping physical `VoltageGate` (and thus `SingleChannel`) objects. It manages named voltage presets (`VoltageTuningPoint` macros) and creates `VoltageSequence` instances.
+
+#### 2.1.2 VoltageSequence
+
+`VoltageSequence` uses the GateSet to apply QUA voltage operations (steps, ramps) within a QUA Program. It tracks channel states, optionally including integrated voltage for DC compensation, which is useful for AC-coupled lines. **One of its primary features is that it keeps track of the current voltage on each channel, allowing you to ramp to absolute voltages even with sticky mode enabled.**
+
+#### 2.1.3 VirtualGateSet
+
+A subclass of `GateSet`. It manages a list of `VirtualisationLayer` objects, which define the transformations from virtual gate voltages to underlying (either physical or lower-level virtual) gate voltages.
+
+#### 2.1.4 VirtualisationLayer
+
+Represents a single linear transformation (matrix) from a set of source (virtual) gates to a set of target gates.
+
+#### 2.1.5 VoltageGate
+
+`VoltageGate` is a QuAM channel built specifically to handle quantum dot and spin qubit experiments. It inherits from `SingleChannel`, adding an `offset_parameter` and `attenuation` values. 
+
+### 2.2 Workflow
+
+**This document will start with an end-to-end example before diving into the specifics. This example workflow takes place in 7 broad steps:**
+
+#### 1.  Define QUAM `VoltageGate` objects for physical gates
+
+- Below is an example of how a `VoltageGate` is instantiated. As appropriate, add `offset_parameter` and `attenuation` arguments. 
+
+  ```python
+  from quam_builder.architecture.quantum_dots.components import VoltageGate
+  from quam.components import StickyChannelAddon, pulses
+
+
+  channel_p1 = VoltageGate(
+    opx_output = ("con1", 1), #Specify the OPX output
+    sticky=StickyChannelAddon(duration=1_000, digital=False),  # For DC offsets
```

Comment:
Here I would explain why the channel must be sticky, because it is a strong requirement for the whole workflow to work.